docs/en_us/platform_api/source/index.rst

```diff
@@ -1,15 +1,21 @@
-################################################
-edX Platform API Version 0.5
-################################################
+##################
+EdX Platform APIs
+##################
 
 .. toctree::
-    :maxdepth: 2
+    :titlesonly:
     
     read_me
     preface
+    change_log
```

Comment:
The change log is very out-of-date (it only has a single entry from January).

http://draft-platform-apis.readthedocs.org/en/latest/change_log.html